cmd/traffic/cmd/manager/cluster/nodewatcher.go

```diff
@@ -60,92 +84,53 @@ func newNodeWatcher(ctx context.Context, lister licorev1.NodeLister, informer ca
 }
 
 func (w *nodeWatcher) changeNotifier(ctx context.Context, updateSubnets func(set subnet.Set)) {
-	// Check for changes every 5 second
-	const nodeReviewPeriod = 5 * time.Second
-
-	// The time we wait from when the first change arrived until we actually do something. This
-	// so that more changes can arrive (hopefully all of them) before everything is recalculated.
-	const nodeCollectTime = 3 * time.Second
+	var lastSent subnet.Set
+	sendUpdate := func() {
+		w.lock.Lock()
+		doSend := !w.subnets.Equals(lastSent)
+		if doSend {
+			lastSent = w.subnets.Clone()
+		}
+		w.lock.Unlock()
+		if doSend {
+			dlog.Debugf(ctx, "nodeWatcher calling updateSubnets with %v", w.subnets)
```

Comment:
in line 94 is release lock and we do a read `w.subnets` here. So we read it without lock. Is it ok?

Should we use lastSent here instead?

```go
dlog.Debugf(ctx, "nodeWatcher calling updateSubnets with %v", lastSent)
```